### DISCUSSION TITLE
如何手动导入UUID？

### DISCUSSION THREAD

**Discussion by ConstasJ** [bs-community/blessing-skin-server#444] [Q&A]
由于我们服务器装载了大量只认UUID的插件和模组，所以为了更加平稳地将服务器的登录方式由正版+离线改成Blessing Skin统一登录，希望能够将已有用户的UUID手动导入Blessing Skin Server。  
但是，官方文档中“手动导入UUID”一节没有内容（是空的）。如果可以的话，能提供一个可行的方案吗？

**Comment by ghost** [comment]
在旧版本的Yggdrasil插件里有手动导入UUID的功能，
你可以尝试使用旧版本的Yggdrasil插件进行测试
